glean_parser/metrics.py

```diff
@@ -240,7 +256,19 @@ class Event(Metric):
 
     @property
     def allowed_extra_keys(self):
-        return list(self.extra_keys.keys())
+        # Sort keys so that output is deterministic
+        return sorted(list(self.extra_keys.keys()))
+
+    @staticmethod
+    def validate_extra_keys(extra_keys, config):
```

Comment:
@mdboom not strictly related to this code location, but should we at least standardize over the `description` metadata key, to have parity with the current behaviour? I mean, we should at the very least require a `description` key in the metadata for a specific `extra_key`. What do you think?

Reply:
It is -- it's enforced by the jsonschema in this PR.  The only reason this validation happens here (in Python) is because it's dependent on a flag, which is something JSON schema can't do.

Reply:
Mh, interesting. For some weird UI artifact this PR was only showing up changes to 2 files to me. I was missing changes to the YAML file.